src/com/tigersale/userInterface/inventoryManager/EditProductView.java

```diff
@@ -150,4 +182,24 @@ private void editAmount(Product product) {
             }
         }
     }
+
+    private void editBrand(Product product) {
```

Comment:
For all of these you get the new value, like the new brand, but you dont update it in the database.  Did you want to do that portion, or did you want me?